javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+var $openBtn = document.querySelector('.modalBtn');
+var $modal = document.querySelector('.modal');
+var $container = document.querySelector('.container');
+
+$openBtn.addEventListener('click', function event() {
```

Comment:
You are naming the function `event` here, but I assume you meant that as a parameter.